### ISSUE TITLE
Disconnection of the Model

### ISSUE THREAD

**Issue by gwroldan** [rsaenen/ngx-input-file#6]
After deleting all the files I go back to find other files but I no longer upload them to the model. The tests I'm doing with the files of this repo.

**Comment by gwroldan** [comment]
The tests I'm doing with the files of this repo.

**Comment by rsaenen** [comment]
I will flag this package to deprecated and make a new component more modern and friendly user. Don't hesitate to contribute to this package too.

**Comment by rsaenen** [comment]
To add another point, the disconnection of the model comes from with using wrong function.
When manipulating the model to add or remove files, the reference  of the array (model) will be loose.
It causes by thing function I guess:
```typescript
/**
     * Remove handler.
     */
    private removeHandler(): void {
        this.dropZoneDisabled = false;
        this.inputFile.nativeElement.value = '';
        this.model = new Array<File>();
    }
```

**Comment by padjon** [comment]
came across this bug recently too.
clearing the array instead of replacing it fixes the disconnection:

`private removeHandler(): void {
        this.dropZoneDisabled = false;
        this.inputFile.nativeElement.value = '';
        this.model.splice(0, this.model.length);
}`

**Comment by padjon** [comment]
Created a pull-request for the issue: #10

**Comment by rsaenen** [comment]
Thanks for contributing.
The splice is the solution to keep the reference and double binding.

Released with 1.0.4
